osbenchmark/metrics.py

```diff
@@ -1550,6 +1550,14 @@ def store_test_execution(self, test_execution):
         with open(self._test_execution_file(), mode="wt", encoding="utf-8") as f:
             f.write(json.dumps(doc, indent=True, ensure_ascii=False))
 
+    def store_aggregated_execution(self, test_execution):
+        doc = test_execution.as_dict()
+        aggregated_execution_path = paths.aggregated_results_root(self.cfg, test_execution_id=test_execution.test_execution_id)
+        io.ensure_dir(aggregated_execution_path)
+        aggregated_file = os.path.join(aggregated_execution_path, "test_execution.json")
```

Comment:
May be better to call this `aggregated_test_execution.json`.

Reply:
+1

Reply:
@gkamat @IanHoang done + I added some changes to metrics.py so OSB can locate the new aggregate folder if you guys wouldn't mind taking a look👍